tests/Microsoft.Identity.Test.Unit/Helpers/CertHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Microsoft.Identity.Test.Common.Core.Helpers
+{
+    internal class CertHelper
+    {
+        public static X509Certificate2 CreateTestCert()
```

Comment:
Can this be cached, so it's only created once per test run? Otherwise, each test creates a new cert each time.